tests/upgrade/customPassword_test.js

```diff
@@ -0,0 +1,79 @@
+
+Data(clientDbServices).Scenario(
+  'Adding custom agent Password, Custom Label before upgrade At service Level @pre-custom-password-upgrade',
+  async ({
+    I, inventoryAPI, current,
+  }) => {
+    const {
+      serviceType, name, upgrade_service,
+    } = current;
+    const {
+      service_id, node_id, address, port,
+    } = await inventoryAPI.getServiceDetailsByPartialName(name);
+
+    const { agent_id: pmm_agent_id } = await inventoryAPI.apiGetPMMAgentInfoByServiceId(service_id);
+    let output;
+
+    switch (serviceType) {
+      case SERVICE_TYPE.MYSQL:
+        output = await I.verifyCommand(
+          `pmm-admin add mysql --node-id=${node_id} --pmm-agent-id=${pmm_agent_id} --port=${port} --password=GRgrO9301RuF --host=${address} --query-source=perfschema --agent-password=uitests --custom-labels="testing=upgrade" upgrade-${upgrade_service}`,
+        );
+        break;
+      case SERVICE_TYPE.POSTGRESQL:
+        output = await I.verifyCommand(
+          `pmm-admin add postgresql --username=pmm --password=pmm --node-id=${node_id} --pmm-agent-id=${pmm_agent_id} --port=${port} --host=${address} --agent-password=uitests --custom-labels="testing=upgrade" upgrade-${upgrade_service}`,
+        );
+        break;
+      case SERVICE_TYPE.MONGODB:
+        output = await I.verifyCommand(
+          `pmm-admin add mongodb --username=pmm_mongodb --password="5M](Q%q/U+YQ<^m" --port=27017 --host=${address} --agent-password=uitests --custom-labels="testing=upgrade" upgrade-${upgrade_service}`,
+        );
+        break;
+      default:
+    }
+  },
+);
+
+Data(clientDbServices).Scenario(
+  'Verify if Agents added with custom password and custom label work as expected Post Upgrade @post-client-upgrade @post-custom-password-upgrade',
+  async ({
+    current, inventoryAPI, grafanaAPI,
+  }) => {
+    const {
+      serviceType, metric, upgrade_service, name,
+    } = current;
+
+    const apiServiceDetails = await inventoryAPI.getServiceDetailsByPartialName(`upgrade-${upgrade_service}`);
+    const { custom_labels } = await inventoryAPI.apiGetNodeInfoByServiceName(serviceType, apiServiceDetails.service_name);
+
+    await grafanaAPI.checkMetricExist(metric, { type: 'service_name', value: apiServiceDetails.service_name });
+    if (serviceType !== SERVICE_TYPE.MYSQL) {
+      assert.ok(custom_labels, `Node Information for ${serviceType} added with ${upgrade_service} is empty, value returned are ${custom_labels}`);
+      assert.ok(custom_labels.testing === 'upgrade', `Custom Labels for ${serviceType} added before upgrade with custom labels, doesn't have the same label post upgrade, value found ${custom_labels}`);
+    }
+  },
+);
+
+Scenario(
+  'PMM-T1189 - verify user is able to change password after upgrade @post-custom-password-upgrade',
+  async ({ I, homePage }) => {
+    const newPass = process.env.NEW_ADMIN_PASSWORD || 'admin1';
```

Comment:
Same here with the hardcoded credentials.

Reply:
This is just password for the test, it is not used for different test.